client/my-sites/checklist/wpcom-checklist/checklist-navigation/index.jsx

```diff
@@ -0,0 +1,104 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Children, Component } from 'react';
+import { connect } from 'react-redux';
+import { find, reduce } from 'lodash';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ProgressBar from 'components/progress-bar';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+export class ChecklistNavigation extends Component {
+	static propTypes = {
+		siteSlug: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+		setStoredTask: PropTypes.func,
+		setNotification: PropTypes.func,
```

Comment:
From the usage, `setStoredTask` and `setNotification` should be `isRequired`. Also, we are missing a few props here, e.g. `closePopover` and `recordTracksEvent`.